com.unity.cinemachine/Editor/Editors/CinemachineBlenderSettingsEditor.cs

```diff
@@ -149,5 +155,100 @@ SerializedProperty element
                     blendProp.FindPropertyRelative(() => def2.Time).floatValue = 2f;
                 };
         }
+#else
+        public override VisualElement CreateInspectorGUI()
+        {
+            var ux = new VisualElement();
+
+            var vcam = Target;
```

Comment:
This is not used. Maybe you meant to use it below at `Target.CustomBlends`.

Reply:
no, it's here by mistake